build/depends.py

```diff
@@ -1143,15 +1143,12 @@ def configure(self, build, conf):
         if build.platform_is_linux or \
                 build.platform_is_bsd:
             mixxx_files = [
-                ('SETTINGS_PATH', '.mixxx/'),
                 ('SETTINGS_FILE', 'mixxx.cfg')]
         elif build.platform_is_osx:
             mixxx_files = [
-                ('SETTINGS_PATH', 'Library/Application Support/Mixxx/'),
```

Comment:
yes, of course. I left it because I was not sure we will not need different settings per os in the near future.